src/index.html

```diff
@@ -1,17 +1,179 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        <label class="form-field">
+        Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          class="input-field"
+          required
+          >
+        </label>
+
+        <label class="form-field">
+         Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          class="input-field"
+          required
+          >
+        </label>
+
+        <label class="form-field">
+         How old are You?
+          <input
+          type="number"
+          name="age"
+           min="1" max="100"
+           value="12"
+           class="input-field"
```

Comment:
you should be consistent about your spacing + each attribute should be moved to a new line
```suggestion
           type="number"
           name="age"
           min="1" 
           max="100"
           value="12"
           class="input-field"
```